alerter.cpp

```diff
@@ -1,32 +1,62 @@
 #include <iostream>
 #include <assert.h>
+#include <string>
 
 int alertFailureCount = 0;
+#define TEMPTHRESHOLD 200.0
+#define SUCCESS 200
+#define FAILURE 500
 
-int networkAlertStub(float celcius) {
+int testNetworkAlertStub(float celcius)
+{
     std::cout << "ALERT: Temperature is " << celcius << " celcius.\n";
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // stub always succeeds and returns 200
-    return 200;
+    if (celcius > TEMPTHRESHOLD)
+    {
+        return FAILURE;
+    }
+    else
+    {
+        return SUCCESS;
+    }
+}
+
+int testProductionAlertStub(float celcius)
+{
+    std::cout << "ALERT: Temperature is " << celcius << " celcius.\n";
+    // Threshold needs to be defined for production environment, currently returning SUCCESS in all cases.

```

Comment:
if this is production code, then please define the threshold and update this code so that test code does test this.